src/components/Select/select.module.scss

```diff
@@ -0,0 +1,37 @@
+.select-wrapper {
+    width: 100%;
+    position: relative;
+
+    .select-dropdown-overlay {
+        width: 400px;
+    }
+    .select-dropdown-main-wrapper {
+        width: 100%;
+    }
+    .select-input {
+        cursor: pointer;
+        min-width: unset !important;
+    }
+
+    .multi-select-pills {
+        position: absolute;
+        z-index: 1;
+        left: 26px;
+        top: 6px;
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+    }
+
+    .multi-select-pill {
```

Comment:
recommend some spacing between neighbor pills, possibly like:
```
.multi-select-pill {
  ...
  + .multi-select-pill {
    margin-left: 0.25rem;
  }
}
```

Reply:
@jhoward-eightfold done added
<img width="260" alt="Screenshot 2022-06-10 at 8 47 17 AM" src="https://user-images.githubusercontent.com/106226099/172983511-f0c382c9-162c-4e3e-8d76-dfa3d3958e29.png">

Reply:
Awesome. thank you. Any thoughts on why these pills are are not the same height? might be an issue with the pill component itself.

Reply:
Good find @jhoward-eightfold. minor css issue. fixed now.